ui/src/reducers/index.js

```diff
@@ -39,6 +65,21 @@ export default (state = initialState, action) => {
           rows: sortedApps
         }
       };
+    case REVERSE_SORT_APP_DETAILS:
+      const newOrder = state.sortBy.direction === SortByDirection.asc ? SortByDirection.desc : SortByDirection.asc;
+      const colIndex = action.payload.index;
+      const newRows = state.appDetailRows.sort(
+        (a, b) => (a[colIndex] < b[colIndex] ? -1 : a[colIndex] > b[colIndex] ? 1 : 0)
+      );
+      const sortedAppDetails = newOrder === SortByDirection.asc ? newRows : newRows.reverse();
```

Comment:
sorting logic and asc/ desc sort is same as REVERSE_SORT above, so will probably abstract it into a seperate method, and call that method here (& in REVERSE_SORT above)